Alignment/OfflineValidation/macros/makeArrowPlots.C

```diff
@@ -39,9 +42,9 @@ void DrawRPhiLegend(double xLim, double yLim, double barrelRPhiRescale)
   float testBlockSize = 0.2*xLim; //5cm in axis unit
   float disty = 0;
   float dYTest =0.1*xLim;
-  float dZTest =2;
-  float xLegObj = 20;
-  float yLegObj = 18;
+  //float dZTest =2;
+  //float xLegObj = 20;
```

Comment:
remove commented coded if not used.

Reply:
Duplicate comment with Tamas. Changed with next commit.